Python/fileutils.c

```diff
@@ -922,6 +922,95 @@ _Py_GetLocaleEncodingObject(void)
     return str;
 }
 
+#ifdef HAVE_NON_UNICODE_WCHAR_T_REPRESENTATION
+
+/* Check whether current locale uses Unicode as internal wchar_t form. */
+int
+_Py_IsLocaleUnicodeWchar(void)
```

Comment:
I'm not sure if "_Py_IsLocaleUnicodeWchar" sentence makes sense in english :-) Maybe "Does Locale Use non-Unicode wchar_t?" => _Py_DoesLocaleUseNonUnicodeWcharT(), or just _Py_LocaleUsesNonUnicodeWcharT()? How lovely is that name? :-D

Reply:
You are right :) - I renamed it with your suggestion (not including the trailing T as other `wchar_t` related functions seem to omit it as well).